tests/rustdoc-ui/doctest/check-attr-test.rs

```diff
@@ -3,27 +3,39 @@
 #![deny(rustdoc::invalid_codeblock_attributes)]
 
 /// foo
+//~^ ERROR unknown attribute `compile-fail`
+//~^^ ERROR unknown attribute `compilefail`
+//~^^^ ERROR unknown attribute `comPile_fail`
```

Comment:
More conventional:

```suggestion
//~^ ERROR unknown attribute `compile-fail`
//~| ERROR unknown attribute `compilefail`
//~| ERROR unknown attribute `comPile_fail`
```

Similarly for the other cases. Or does it not work for some peculiar reason?